modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -63,11 +63,12 @@ When the Operator installation is finished, the Manila CSI driver is deployed on
 .Verification steps
 . Verify that the ManilaDriver CR was created successfully by entering the following command:
 +
+[source,terminal]
 ----
 $ oc get all -n openshift-manila-csi-driver
 ----
 +
-Example output:
+.Example output
```

Comment:
Should there also be a `[source,terminal]` here?

Reply:
Yes, good catch! Resolved.